security/advancedtls/examples/credential_reloading_from_files/client/main.go

```diff
@@ -0,0 +1,97 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// The client demonstrates how to use the credential reloading feature in
+// advancedtls to make a mTLS connection to the server.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	"google.golang.org/grpc"
+	pb "google.golang.org/grpc/examples/helloworld/helloworld"
+	"google.golang.org/grpc/security/advancedtls"
+	"google.golang.org/grpc/security/advancedtls/testdata"
+)
+
+var address = "localhost:50051"
+
+const (
+	// Default timeout for normal connections.
+	defaultConnTimeout = 5 * time.Second
+	// Intervals that set to monitor the credential updates.
+	credRefreshingInterval = 200 * time.Millisecond
```

Comment:
Does this have to be this low? Or at least add a comment saying this is set to an impractically low value just for this example, and that in real use cases, this should be in the order of minutes.

Reply:
Sounds good! It doesn't need to be this low. I've updated it to 1 minute.